pennylane_lightning/src/simulators/lightning_kokkos/gates/tests/Test_StateVectorKokkos_NonParam.cpp

```diff
@@ -75,7 +75,7 @@ TEMPLATE_TEST_CASE("StateVectorKokkos::applyHadamard",
         SECTION("Apply directly") {
```

Comment:
These are no longer falling under the `Apply directly` signature, since we dynamically look them up. Same with above changes.

Reply:
Good point, we can remove a lot of fat here.